warden/x/act/keeper/actions_keeper.go

```diff
@@ -56,20 +56,46 @@ func (k ActionKeeper) New(ctx context.Context, action *types.Action) (uint64, er
 		return 0, err
 	}
 
+	err = k.setActionsForMentions(ctx, action, id)
+	if err != nil {
+		return 0, err
+	}
+
+	return id, nil
+}
+
+func (k *ActionKeeper) setActionsForMentions(ctx context.Context, action *types.Action, id uint64) error {
```

Comment:
it's a minor thing but maybe a name like this is slighly clearer

```suggestion
func (k *ActionKeeper) updateMentions(ctx context.Context, action *types.Action, id uint64) error {
```

Reply:
I agree, also I don't think that's minor, thanks!